runelite-client/src/main/java/net/runelite/client/plugins/banktags/BankTagsPlugin.java

```diff
@@ -183,7 +183,7 @@ public void onScriptEvent(ScriptCallbackEvent event)
 
 				List<String> tags = Arrays.asList(tagsConfig.split(","));
 
-				if (tags.stream().anyMatch(tag -> tag.contains(search)))
+				if (tags.stream().anyMatch(tag -> tag.toLowerCase().contains(search)))
```

Comment:
Uh, sorry for nit-picking, but lowercasing the initial tagConfig string before splitting would be more efficient :d Also i would probably lowercase search too, just in case

Reply:
What makes `search` always lowercase?